clang/test/CodeGen/X86/avx10_2_512minmax-error.c

```diff
@@ -1,7 +1,7 @@
 // RUN: %clang_cc1 %s -flax-vector-conversions=none -ffreestanding -triple=x86_64 -target-feature +avx10.2-512 \
-// RUN: -Wno-invalid-feature-combination -emit-llvm -verify
+// RUN: -Wno-invalid-feature-combination -emit-llvm-only -verify
```

Comment:
I just realized `-emit-llvm is not required here. You can directly removed it. Sorry for mistake.